core/win32/inject.c

```diff
@@ -259,6 +259,7 @@ inject_into_thread(HANDLE phandle, CONTEXT *cxt, HANDLE thandle, char *dynamo_pa
             /* skip xmm slots */
             bufptr += MCXT_TOTAL_SIMD_SLOTS_SIZE / sizeof(*bufptr);
         }
+        bufptr += MCXT_TOTAL_OPMASK_SLOTS_SIZE / sizeof(*bufptr);
```

Comment:
But we need to copy the CONTEXT opmask regs.  I would expect a FIXME/TODO comment if this is going to be in a future CL.  Not a good state to leave the code in with just an adjustment but uninitialized fields and no explanation.

Reply:
That's right. Ok, comment added.